libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -694,6 +694,18 @@ bool ASTJsonConverter::visit(IndexAccess const& _node)
 	return false;
 }
 
+bool ASTJsonConverter::visit(IndexRangeAccess const& _node)
+{
+	std::vector<pair<string, Json::Value>> attributes = {
+		make_pair("baseExpression", toJson(_node.baseExpression())),
```

Comment:
Do need `..Expression` for the three of these?

Reply:
Nope, but it's consistent with ``IndexAccess`` this way, that's the reason... we could change both, though?

Reply:
then it's fine!